services/core-web/src/components/mine/Permit/PermitConditions.tsx

```diff
@@ -76,17 +86,16 @@ const PermitConditions: FC<PermitConditionProps> = ({ latestAmendment }) => {
     return Promise.resolve();
   };
 
-  if (fetchingPermits) {
+  if (isLoading) {
```

Comment:
🤨 Didn't like that name?

Reply:
Basically it's because of something I added and then took out! And then I decided that I liked "isLoading" anyway and left it after I took it out.

If you're curious, the call to fetchPermits on the parent when the extraction was complete wasn't getting called properly at first (and so I didn't notice it was there) so I made a call _here_ to fetchPermitConditions and isLoading also was based on that. But that created a whole data issue because it wasn't stored in latestAmendment.permit_conditions where everything was looking for it.

Reply:
Ah, cool!